blocks/button/button.css

```diff
@@ -0,0 +1,8 @@
+.button{
```

Comment:
done ✅

Reply:
@MarPostovik I just noticed something else that could be improved here 🙂 

Let's set the CSS `cursor` property with the value of `pointer`. This property is vital because it enhances user experience by providing visual feedback to users, indicating that the button is clickable. This small addition can significantly improve usability and make our buttons more intuitive for users to interact with.

Reply:
done ✅